cmd/codegen/generator/go/templates/module_funcs.go

```diff
@@ -14,7 +14,7 @@ import (
 const errorTypeName = "error"
 
 var voidDef = Qual("dag", "TypeDef").Call().
-	Dot("WithKind").Call(Id("dagger").Dot("VoidKind")).
+	Dot("WithKind").Call(Id("dagger").Dot("TypeDefKindVoidKind")).
```

Comment:
So this is annoying - the name now stutters for some of our core types (but I guess this has always been the case in typescript/python).

Ideally we should also have the value here just be `VOID` instead of `VOID_KIND`? But this would be a pretty big backwards incompat change.

We could do this with the rest of the enum changes we're planning?